### DISCUSSION TITLE
Playtime: When transport sync is "Full" and playing clip from stopped state, also start REAPER transport

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1228] [Ideas]
Boring/pain to implement right now. At the moment, I don't consider this as essential for the final release. Might rather disable the "Transport sync" feature for now.

**Comment by atencion** [comment]
Please don`t disable the transport sync.  That would really mess the whole workflow and set PLaytime as an island inside Reaper.

**Comment by helgoboss** [comment]
Ok, then I'll just leave it as it is for now.
